modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -96,6 +96,11 @@ class Session implements ISession {
 
     final List<Closure> igniters = new ArrayList<>(20)
 
+    /**
+     * Public custom fsync on a file path since some implementations of `sync` in slim distros might not support fsyncing on a path to push file updates to the shared dir
+     */
+    final String CUSTOM_FSYNC_URL = "https://latch-public.s3.us-west-2.amazonaws.com/nextflow-v2/custom_fsync"
```

Comment:
nit: i dont think we use all caps for any globals so prob just change to `customer_fsync_url`